app/services/reports/docx.rb

```diff
@@ -81,6 +86,88 @@ def self.render_img_element(docx, element, options = {})
       align style[:align] || :left
     end
   end
-end
 
+  def self.render_list_element(docx, element, options = {})
+    bookmark_items = Reports::Docx.recursive_list_items_renderer(docx, element)
+
+    bookmark_items.each_with_index do |(key, item), index|
+      if item[:type] == 'image'
+        docx.bookmark_start id: index, name: key
+        docx.p do
+          br
```

Comment:
Why do you add a new line here? I think the `p` element already create newline

Reply:
To have some top margin